oaknational/Oak-Web-Application
# Feat/lesq 697/update seo [LESQ-697]

## Description

Music year: 1964

[ticket](https://www.notion.so/oaknationalacademy/I-want-to-ensure-the-SEO-meta-data-is-correct-ee7db2913e1b4bbd94ca5b7b4987e3af?pvs=4)

- Update SEO for lesson journey pages according to the ticket and spreadsheet requirements
- There were a few cases where I deviated slightly from the spreadsheet, see the ticket for details
- I could not see the incorrect pagination so have left as it is, if you can reproduce please let me know

## How to test

1. Go to https://deploy-preview-2315--oak-web-application.netlify.thenational.academy
2. Click on the lesson journey
3. You should see correct title, description and canonical tags on pages
